tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions.dart

```diff
@@ -304,6 +296,20 @@ class Restrictions {
       ];
     }
 
+    var currentModel =
+        parsedModels.findByTableName(tableName) as ClassDefinition;
+
+    var ancestorWithTable = _findOtherTableClassInHierarchy(currentModel);
+
+    if (ancestorWithTable != null) {
+      return [
+        SourceSpanSeverityException(
+          '${currentModel.className} cannot have a table, another class in its hierarchy already declares a table "${ancestorWithTable.className}".',
```

Comment:
Discussion: I would like to give better instructions for the action to take to fix this issue.

This is validation on the table name right? So we will be in the context of the file.


For example: `The "table" property is not allowed because another class, "${ancestorWithTable.className}", in the class hierarchy already has one defined. Only one table definition is allowed when using inheritance.`